examples/tf/rl2_ppo_ml45.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+"""Example script to run RL2 in ML45."""
+# pylint: disable=wrong-import-order
+from metaworld.benchmarks import ML45
+import numpy as np
+
+from garage import wrap_experiment
+from garage.experiment import task_sampler
+from garage.experiment.deterministic import set_seed
+from garage.np.baselines import LinearFeatureBaseline
+from garage.sampler import LocalSampler
+from garage.tf.algos import RL2PPO
+from garage.tf.algos.rl2 import RL2Env
+from garage.tf.algos.rl2 import RL2Worker
+from garage.tf.experiment import LocalTFRunner
+from garage.tf.policies import GaussianGRUPolicy
+
+
+@wrap_experiment
+def rl2_ppo_ml45(ctxt=None, seed=1):
+    """Train PPO with ML45 environment.
+
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        seed (int): Used to seed the random number generator to produce
+            determinism.
+
+    """
+    set_seed(seed)
+    with LocalTFRunner(snapshot_config=ctxt) as runner:
+        max_path_length = 100
```

Comment:
FYI this is usually 150 for metaworld, but I guess go with whatever you used for ICML